src/explicit_instantiation_macros.h

```diff
@@ -228,7 +228,7 @@ enum CompressionType {
     IF(BOOL(NO_GPU))      (FLOAT_LOOP_ALL(INIT_TEMPLATES, false))
 
 #define INIT_ALL(INIT_TEMPLATES) \
-    DEVICE_LOOP_ALL(INIT_TEMPLATES)
+    __device___LOOP_ALL(INIT_TEMPLATES)
```

Comment:
This will fail!

Reply:
this is a find-replace error. I didn't find it yet because I didn't compile everything again after the latest commit